libmamba/include/mamba/core/util_random.hpp

```diff
@@ -42,12 +45,12 @@ namespace mamba
 
     inline std::string generate_random_alphanumeric_string(std::size_t len)
     {
-        static constexpr auto chars = "0123456789"
-                                      "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
-                                      "abcdefghijklmnopqrstuvwxyz";
+        static constexpr auto chars = std::string_view("0123456789"
```

Comment:
I am slightly against this change because I don't really see any benefit and it will likely compile a little bit slower (especially also with the additional includes :)